code/modules/research/designs/autolathe_designs.dm

```diff
@@ -949,3 +949,11 @@
 	materials = list(MAT_METAL = 1500, MAT_GLASS = 500)
 	build_path = /obj/item/mail_scanner
 	category = list("initial", "Miscellaneous")
+
+/datum/design/spraybottle
+	name = "Spray Bottle"
+	id = "spraybottle"
+	build_type = AUTOLATHE
+	materials = list(MAT_METAL = 500, MAT_GLASS = 2000)
```

Comment:
the metal cost on this should be way higher for balance reasons tbh

Reply:
I would add plastic :D

Reply:
> the metal cost on this should be way higher for balance reasons tbh

How much more?

Reply:
> I would add plastic :D

Add plastic? Or make in crafable only from plastic?